cmd/svcat/plan/get_cmd.go

```diff
@@ -169,19 +169,21 @@ func (c *getCmd) get() error {
 		plan, err = c.App.RetrievePlanByClassAndPlanNames(c.className, c.name)
 
 	default:
-		plan, err = c.App.RetrievePlanByName(c.name)
+		plan, err = c.App.RetrievePlanByName(c.name, servicecatalog.ScopeOptions{
+			Scope: servicecatalog.ClusterScope,
+		})
 
 	}
 	if err != nil {
 		return err
 	}
 	// Retrieve the class as well because plans don't have the external class name
-	class, err := c.App.RetrieveClassByID(plan.Spec.ClusterServiceClassRef.Name)
+	class, err := c.App.RetrieveClassByID(plan.GetClassID())
 	if err != nil {
 		return err
 	}
-
-	output.WritePlan(c.Output, c.OutputFormat, *plan, *class)
+	csp := plan.(*v1beta1.ClusterServicePlan)
```

Comment:
Same here, let's leave a comment explaining that describe hasn't been implemented yet for namespaced plans and link to that issue.